playbooks/README.md

```diff
@@ -0,0 +1,6 @@
+# Playbook names
+
+This repository contains playbooks to use in the deployment mechanism.
+
+To understand how they are structured, they are named in a similar fashion:
+```<DEPLOYMENT_MECHANISM|generic>-<action>.yml```
```

Comment:
Im not sure if I agree with this. Are we gonna have different playbooks for kvm/openstack/vagrant/docker/local here? Will they be split? Why not go with the actual status (deploy_caasp.yml, deploy_ses_aio.yml, etc...) and when we reach a point in which we have something new(lets say KVM deployment instead of openstack) we discuss about how to tackle that? It could be new playbooks or it could be in the same playbooks with different tasks for example, so renaming things around this soon when its not needed feels a bit weird to me.

Reply:
We just keep the same 'spirit' of the previous folder structure. Before that, anyone could implement it's own alternative "step", and load that from its deployment mechanism shell script. Here it would be the same. I just expect generic playbooks to be re-used.

Reply:
But I am open to other suggestions :)